Cargo.toml

```diff
@@ -25,6 +25,13 @@ mitosis = {version = "0.1.1", optional = true}
 log4rs = "1.3.0"
 bdk = "0.29.0"
 
+bdk_wallet= {git = "https://github.com/bitcoindevkit/bdk", subdir="crates/wallet",  rev = "3b040a7ee69c7fefac6c3ab57c17b220e0beb9fb"}
+bdk_persist= {git = "https://github.com/bitcoindevkit/bdk", subdir="crates/bdk_persist",  rev = "3b040a7ee69c7fefac6c3ab57c17b220e0beb9fb"}
+bdk_chain= {git = "https://github.com/bitcoindevkit/bdk", subdir="crates/bdk_chain",  rev = "3b040a7ee69c7fefac6c3ab57c17b220e0beb9fb"}
+
+strum_macros = "0.26.4"
+strum = "0.26.2"
```

Comment:
Can we work around without these? I am reluctant to add extra utility deps if we can just do it with few more lines of code.